src/containers/guest-home-page/login-form/LoginForm.jsx

```diff
@@ -31,6 +35,17 @@ const LoginForm = ({
     openModal({ component: <ForgotPassword /> })
   }
 
+  useEffect(() => {
```

Comment:
use useMemo instead useEffect and useState

Reply:
This is really a better solution, thanks for your advice, implemented it